File: src/utilpc/Theme.java
package utilpc;

import java.awt.Color;

public class Theme {
	public static final ColorSet LIGHT = new ColorSet(
			new Color(216, 216, 216),
			new Color(179, 229, 252), //Tooltip BG
			new Color(144, 144, 144), //Button, Progress bar Color
			new Color(248, 187, 0),
			new Color(189, 189, 189), //Disabled Text
			new Color(115, 164, 209),
			new Color(174, 213, 129),
			new Color(129, 212, 250),
			new Color(245, 245, 245),
			new Color(255, 204, 128),
			new Color(239, 154, 154),
			new Color(91, 91, 91),
			new Color(128, 222, 234),
			new Color(91, 91, 91)
			);

	public static final ColorSet DARK = new ColorSet(
			new Color(64, 64, 64),
			new Color(84, 110, 122), //Tooltip BG
			new Color(16, 16, 16), //Button, Progress bar Color
			new Color(248, 187, 0),
			new Color(174, 174, 174), //Disabled Text
			new Color(115, 164, 209),
			new Color(174, 213, 129),
			new Color(129, 212, 250),
			new Color(64, 64, 64), //Panel BG
			new Color(255, 204, 128),
			new Color(239, 154, 154),
			new Color(245, 245, 245), //Text Color
			new Color(98, 117, 127),
			new Color(245, 245, 245) //Text Color
			);
}

File: src/utilpc/Theme.kt
package utilpc

import common.io.assets.Admin
import common.io.assets.Admin.StaticPermitted
import common.io.assets.AssetLoader
import common.io.assets.AssetLoader.AssetHeader
import common.io.assets.AssetLoader.AssetHeader.AssetEntry
import common.io.json.JsonEncoder
import common.io.json.Test
import common.io.json.Test.JsonTest_0.JsonD
import common.io.json.Test.JsonTest_2
import common.pack.Source.AnimLoader
import common.pack.Source.ResourceLocation
import common.pack.Source.SourceAnimLoader
import common.pack.Source.SourceAnimSaver
import common.pack.Source.Workspace
import common.pack.Source.ZipSource
import common.util.stage.EStage
import common.util.stage.StageMap
import common.util.stage.StageMap.StageMapInfo
import common.util.unit.UnitLevel
import io.BCPlayer
import page.JL
import page.anim.AnimBox
import page.support.ListJtfPolicy
import page.support.SortTable
import page.view.ViewBox
import page.view.ViewBox.Conf
import page.view.ViewBox.Controller
import page.view.ViewBox.VBExporter
import java.awt.Color

object Theme {
    val LIGHT: ColorSet = ColorSet(Color(216, 216, 216), Color(179, 229, 252),  // Tooltip BG
            Color(144, 144, 144),  // Button, Progress bar Color
            Color(248, 187, 0), Color(189, 189, 189),  // Disabled Text
            Color(115, 164, 209), Color(174, 213, 129), Color(187, 222, 251), Color(245, 245, 245),
            Color(255, 204, 128), Color(239, 154, 154), Color(91, 91, 91), Color(187, 222, 251),
            Color(91, 91, 91))
    val DARK: ColorSet = ColorSet(Color(64, 64, 64), Color(98, 117, 127),  // Tooltip BG
            Color(16, 16, 16),  // Button, Progress bar Color
            Color(248, 187, 0), Color(174, 174, 174),  // Disabled Text
            Color(98, 117, 127), Color(174, 213, 129), Color(98, 117, 127), Color(64, 64, 64),  // Panel
            // BG
            Color(255, 204, 128), Color(239, 154, 154), Color(245, 245, 245),  // Text Color
            Color(98, 117, 127), Color(245, 245, 245) // Text Color
    )
}
